docs/source/en/glossary.md

```diff
@@ -340,6 +345,12 @@ A pipeline in 🤗 Transformers is an abstraction referring to a series of steps
 
 For more details, see [Pipelines for inference](https://huggingface.co/docs/transformers/pipeline_tutorial).
 
+### PipelineParallel (PP)
```

Comment:
would be nice to have a scheme for each one of these ! Not a must but usually makes things clearer!

Reply:
Added links to the `perf_train_gpu_many` to avoid duplicating content.